templates/project_profile.html

```diff
@@ -306,7 +324,7 @@ <h2>Evaluation</h2>
 {#                            <div class="sending_status_ready" style="display: inline-flex">&#9989;</div>#}
                         {% endif %}
                         <div style="display: inline-flex">
-                            &nbsp&nbsp&nbspSend with Score:
+                            &nbsp&nbsp&nbsp&nbsp&nbsp&nbspSend with Score:
```

Comment:
is this nbsp really how we want to achieve a positioning of this element? Seems hacky. Is there a way to get what we want with the right style/CSS?